src/subgraph/positions/subgraphQueries.ts

```diff
@@ -0,0 +1,39 @@
+import { gql } from "graphql-request";
+
+export const fetchPositionsByUserQuery = (userAddress: string, count: number = 10) =>
```

Comment:
`count` param not used. I guess it should be in `first:${count}`

Reply:
Fixed